benchmarks/pom.xml

```diff
@@ -28,10 +28,10 @@
         <version>2.3.0.Beta2-SNAPSHOT</version>
     </parent>
 
+    <groupId>io.undertow</groupId>
     <artifactId>undertow-benchmarks</artifactId>
     <version>2.3.0.Beta2-SNAPSHOT</version>
-
-    <name>Undertow Benchmarks</name>
+    <name>Undertow Benchmarks - Jakarta EE Variant</name>
```

Comment:
I don't see why adding Jakarta EE variant to the pom descriptions, in the future this will seem dated. If the whole project is just Jakarta compatible, we can assume that each module is Jakarta EE compatible

Reply:
Fixed @fl4via . I also created JIRA for this PR.